spec/forms/openid_connect_authorize_form_spec.rb

```diff
@@ -356,6 +356,65 @@
     end
   end
 
+  describe '#aal' do
+    context 'when AAL1 passed' do
```

Comment:
**suggestion:** I don't think we need to test for this since we don't support it

Reply:
Actually, this should probably be replaced with a test for the default AAL specification instead of AAL1

Reply:
Is there any harm in keeping checking 1? A couple extra cycles, and an extra second per test run?

Reply:
nah not really, other than the setup where you modify the allowed aal values feeling a bit weird to me. Either way is fine by me!